context/app/api/vitessce_confs/paths.py

```diff
@@ -7,6 +7,6 @@
 STITCHED_IMAGE_DIR = "stitched/expressions"
 SPRM_JSON_DIR = "output_json"
 TILE_REGEX = r"R\d+_X\d+_Y\d+"
-STITCHED_REGEX = r"reg\d+_stitched_expressions"
+STITCHED_REGEX = r"reg\d+"
```

Comment:
Sorry not to have the whole picture, but `'_stitched_expressions'` just is moved downstream, right? I'm not sure if other paths also use `STITCHED_REGEX`, but that might be something to make sure of.

Reply:
Yes - this is now just like `TILE_REGEX` above it.  I can add a note about this.